lib/login-providers/browser.js

```diff
@@ -76,7 +76,8 @@ class BrowserLoginProvider extends LoginProviderBase {
             reject(body.error_description);
             return;
           }
-          resolve(super.storeJWT(body.id_token));
+          // Must use this.storeJWT instead of super.storeJWT to support bug in Node 4.
+          resolve(this.storeJWT(body.id_token));
```

Comment:
This is interesting. I thought Node 4.x supported `super` in methods. At least it does according to http://node.green/

Did you change this for some other reason or is this definitely breaking tests?